examples/seismic/tti/operators.py

```diff
@@ -28,14 +28,26 @@ def second_order_stencil(model, u, v, H0, Hz):
 
 def Gzz_centered(field, costheta, sintheta, cosphi, sinphi, space_order):
     """
-    3D rotated second order derivative in the direction z
-    :param field: symbolic data whose derivative we are computing
-    :param costheta: cosine of the tilt angle
-    :param sintheta:  sine of the tilt angle
-    :param cosphi: cosine of the azymuth angle
-    :param sinphi: sine of the azymuth angle
-    :param space_order: discretization order
-    :return: rotated second order derivative wrt z
+    3D rotated second order derivative in the direction z.
+
+    Parameters
+    ----------
+    field : Function
+        Symbolic data whose derivative we are computing.
```

Comment:
"Input for which the derivative is computed"

Reply:
fixed